src/test/java/com/trycloud/pages/DeletedFilesPage.java

```diff
@@ -67,15 +67,57 @@ public WebElement deletedFileName(String FileName){
     }
     //method returns dates in hours in order to sort dates
     public int convertToHours(String date){
+        String empty="";
         if(date.contains("seconds")){
-            return Integer.parseInt(date.replace("seconds ago","1").trim())/3600;
+            //assign modified date
+            empty=date.replace("seconds ago","").trim();
+            //if empty is empty
+            if (empty.isEmpty()){
+                return 0;
+            }else {
+                //if not then
+                return Integer.parseInt(empty)/3600;
+            }
+
         }else if (date.contains("days")){
             return Integer.parseInt(date.replace("days ago","").trim())*24;
         } else if(date.contains("hours")){
             return Integer.parseInt(date.replace("hours ago","").trim());
+        } else if (date.contains("in a few")) {
```

Comment:
repetitive code segment
This can be incorporated into if condition from line 71

            empty=date.replace("seconds ago","").trim();
            empty=date.replace("in a few seconds","").trim();
            //if empty is empty
            if (empty.isEmpty()){
                return 0;
            }else {
                //if not then
                return Integer.parseInt(empty)/3600;
            }

Reply:
Any suggestions?

Reply:
```
        empty = date;
        empty = empty.replace("seconds ago","").trim();
        empty=empty.replace("in a few seconds","").trim();
        //if empty is empty
        if (empty.isEmpty()){
            return 0;
        }else {
            //if not then
            return Integer.parseInt(empty)/3600;
        }
```
delete code from line 86 to 92
delete line 73
add above code to line 73

if replace method cannot find matching target, it returns the original string
therefore, stating it twice will not have affect, if it matches then the content will change. or you can go with if condition